ruby/vampires.rb

```diff
@@ -0,0 +1,74 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+employees.times do |x|
```

Comment:
Great use of the times loop, fwiw the block of |x| is optional if you aren't leveraging each integer.